common/src/test/java/tachyon/util/FormatUtilsTest.java

```diff
@@ -95,6 +97,32 @@ public TestCase(String expected, ByteBuffer input) {
   }
 
   /**
+   * Tests the {@link FormatUtils#byteArrayToHexString(byte[])} method.
+   */
+  @Test
+  public void byteArrayToHexStringTest() {
+    class TestCase {
```

Comment:
this seems like overkill, do you think we'll want to add a lot more test cases? Even now the third case doesn't seem to be testing anything that the other three don't already cover. Maybe we could just do 3-4 asserts?